temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
```

Comment:
It would be beneficial to add a docstring to the `reset_files` function to explain its purpose, arguments, and behavior. [medium]